src/Symfony/Component/Finder/Finder.php

```diff
@@ -85,6 +85,18 @@ public function __construct()
     }
 
     /**
+     * Get lookup directories
+     *
+     * @return array List of directories
+     *
+     * @api
+     */
+    public static function getDirs()
```

Comment:
`$dirs` is an internal property and we shouldn't expose it until we find a good use case supporting this change. The one given in #13822 doesn't seem to be a good justification or perhaps it wasn't explained well.